build.sh

```diff
@@ -5,9 +5,9 @@
 cd $(dirname $0)
 
 WASM_TARGET=wasm32-unknown-unknown
-NIGHTLY=nightly-2021-02-11
+NIGHTLY=$(cat rust-toolchain)
 
-rustup target add ${WASM_TARGET}
+rustup target add ${WASM_TARGET} --toolchain ${NIGHTLY}
```

Comment:
Can remove all of this wasm target and nightly stuff by using a `rust-toolchain.toml`

Reply:
@chinedufn removed